<launch>
  <rosparam command="load" file="$(find openag_brain)/launch/personal_food_computer_v2.yaml" />
  <rosparam command="load" file="$(find openag_brain)/launch/personal_food_computer_var_types.yaml" ns="var_types"/>
  <group ns="environments/test">
    <node pkg="openag_brain" type="recipe_handler.py" name="recipe_handler" />
    <node pkg="openag_brain" type="recipe_persistence.py" name="recipe_persistence" />
  </group>
</launch>
